.github/workflows/auto-publish.yml

```diff
@@ -14,4 +16,11 @@ jobs:
       - uses: actions/checkout@v2
       - uses: w3c/spec-prod@v2 # use the action
         with:
+          W3C_WG_DECISION_URL: "https://lists.w3.org/Archives/Public/public-webrtc/2016Mar/0031.html"
+          W3C_NOTIFICATIONS_CC: "dom@w3.org"
           VALIDATE_LINKS: false
+          TOOLCHAIN: respec
+          W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
+          W3C_BUILD_OVERRIDE: |
+            shortName: mediacapture-region
+            specStatus: WD
```

Comment:
Should this be FPWD or WD?

Reply:
this is for future updates to the document - once the document has been published as FPWD, all future updates are simple WD (until we reach Candidate Recommendation)